aws-lambda-java-runtime-interface-client/src/main/resources/META-INF/native-image/reflect-config.json

```diff
@@ -0,0 +1,38 @@
+[
+{
+  "name":"com.amazonaws.lambda.thirdparty.com.fasterxml.jackson.databind.deser.Deserializers[]"
+},
+{
+  "name":"com.amazonaws.lambda.thirdparty.com.fasterxml.jackson.databind.ext.Java7SupportImpl",
+  "methods":[{"name":"<init>","parameterTypes":[] }]
+},
+{
+  "name":"com.amazonaws.services.lambda.runtime.LambdaRuntime",
+  "fields":[{"name":"logger"}]
+},
+{
+  "name":"example.App",
```

Comment:
I'm guessing this is a mistake?

Reply:
yes sure sorry... only related to my example, forgot it.